### ISSUE TITLE
Suggestion ( all Minecraft versions, especially 1.12.2 )

### ISSUE THREAD

**Issue by Kasia1403** [MinecraftForge/MinecraftForge#9255]
To add estimated time until modpack will be loaded. I really need it. Could You add it, please ?

**Comment by TelepathicGrunt** [comment]
@Kasia1403 this is not possible to add as Forge cannot read the future of what mods could even do that can take up load times and have to take into account all the user’s hardware and software resources. Any predictive timing would be inaccurate. You would be better off launching the modpack yourself and timing it manually. Then you know how long the modpack takes.

**Comment by Kasia1403** [comment]
Do You know Waiting Time mod ? It shows only how long Minecraft loads. But I wanted something that also shows how much time is left until modpack will be loaded. For example if a modpack loads 1 hour, Minecraft loading screen will show 1 hour when it is 0 % loaded and then when it will be 50 % loaded it will show 30 minutes and so on. But I don't know if it is possible to do something like that.

**Comment by ChampionAsh5357** [comment]
As TelepathicGrunt said, any predictive timing on how long it takes the game to load would be inaccurate and involve guesswork. Add to the fact that supported versions pretty much run most of the mod loading process in parallel, it makes zero sense to provide such an estimate.

As for the Waiting Time mod you mentioned, the closest I could find was one where you play pong while the game is loading, so I have no idea what you are trying to say with this.

Regardless of these points though, Forge has mentioned I believe back in 1.13 that it wouldn't be adding this feature in subsequent versions, and I will close this issue because of that.

**Comment by TelepathicGrunt** [comment]
@Kasia1403 How would forge know that x mod has bad code that will take 15 minutes to load if that mod has not stated that before? How would Forge even know the modpack is suppose to take X amount of time to load when it has no prior info or stats about it? 

And again, this is also extremely dependent on the user's hardware and software resources. Running a modpack with 3GB of RAM will take an extremely long time or maybe crash with out of memory error. Someone running with 2 core CPU is going to take longer to load than someone with 4 cores with Forge's multithreading modloading. Even the kind of CPU is a factor. Converting the % to straight time will never be accurate. And there can be no accurate statistic. 

It'll be like the windows estimated time meme of inaccuracy. Again, it's better for you to manually time the modpacks yourself and keep track how long it takes to load them on your specific machine and mods and settings. This feature will never be added to Forge because of all the technical issues and that it is out of scope of Forge
![image](https://user-images.githubusercontent.com/40846040/209472367-889df330-6122-48b6-924f-908c29bb11ff.png)

**Comment by UndefinedBHVR** [comment]
https://www.youtube.com/watch?v=iZnLZFRylbs Here, have a great video by a great content creator explaining why this is not possible, especially not in an accurate way.

**Comment by Kasia1403** [comment]
May I know which launcher is better ? MultiMC or Prism or PolyMC ?
